packages/dcos-integration-test/extra/test_composition.py

```diff
@@ -168,9 +168,11 @@ def test_systemd_units_are_healthy(dcos_api_session) -> None:
         'dcos-telegraf.socket',
         'dcos-fluent-bit.service']
     slave_units = [
+        'dcos-calico-libnetwork-plugin.service',
```

Comment:
several tests defined in [test_clusterstate](https://github.com/mesosphere/dcos-enterprise/blob/master/packages/dcos-integration-test/extra/test_clusterstate.py) under dcos-enterprise repo also need our care.

Reply:
We may add an additional commit to fix this based on the ones pushed by mergebot.

Reply:
Oh I thought this list was appended to the enterprise component. I didn't notice that we have a separate list for the EE components. Will check now 👍